netlify-functions/src/main/scala/feral/netlify/facade/Context.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify.facade
+
+import scala.scalajs.js
+import scala.scalajs.js._
+
+@js.native
+private[netlify] sealed trait Context extends js.Object {
+  def callbackWaitsForEmptyEventLoop: Boolean = js.native
+  def functionName: String = js.native
+  def functionVersion: String = js.native
+  def invokedFunctionArn: String = js.native
+  def memoryLimitInMB: Int = js.native
+  def awsRequestId: String = js.native
+  def logGroupName: String = js.native
+  def logStreamName: String = js.native
+  def identity: UndefOr[Null | js.Object] = js.native
+  def clientContext: UndefOr[Null | js.Object] = js.native
```

Comment:
```suggestion
  def identity: UndefOr[js.Object] = js.native
  def clientContext: UndefOr[js.Object] = js.native
```

See https://github.com/scala-js/scala-js-dom/issues/481#issuecomment-898192431

Reply:
hmm so that'll just be a null, I see